packages/common-ui-web/src/components/VehiclePartSelection/VehiclePartSelection.tsx

```diff
@@ -0,0 +1,85 @@
+import { PartSelectionOrientation, VehiclePart, VehicleType } from '@monkvision/types';
+import { useEffect, useState } from 'react';
+import { useMonkTheme } from '@monkvision/common';
+import { Icon } from '../../icons';
+import { VehicleDynamicWireframe, VehicleDynamicWireframeProps } from '../VehicleDynamicWireframe';
+import { styles } from './VehiclePartSelection.style';
+
+export interface VehiclePartSelectionProps {
```

Comment:
Add TSDoc for this interface : `Props accepted by the VehiclePartSelection component`